CODE-OF-CONDUCT.md

```diff
@@ -0,0 +1,4 @@
+# Code of Conduct
+
+This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct)
```

Comment:
Because this is a .NET project, we should reference the .NET Foundation CoC (The text is the same): [.NET Foundation Code of Conduct](https://dotnetfoundation.org/code-of-conduct).